astacus/coordinator/plugins/clickhouse/steps.py

```diff
@@ -204,7 +204,12 @@ async def run_step(self, cluster: Cluster, context: StepsContext) -> Sequence[Ke
             for child in children:
                 keeper_map_table_path = os.path.join(self.keeper_map_path_prefix, child)
                 data_path = os.path.join(keeper_map_table_path, "data")
-                data = await connection.get_children_with_data(data_path)
+                try:
+                    data = await connection.get_children_with_data(data_path)
+                except NoNodeError:
+                    logger.warning("ZNode %s is missing, table was dropped.  Skipping", data_path)
```

Comment:
It it is just dropped data, probably it should be info or debut level log, nothing to worry about.